src/app/core/entity-components/entity-select/entity-select/entity-select.component.ts

```diff
@@ -172,6 +143,26 @@ export class EntitySelectComponent<E extends Entity> implements OnChanges {
    * has no name, this filters for the entity's id.
    */
   @Input() accessor: accessorFn<E> = (e) => e["name"] || e.getId();
+  /**
+   * The type of entity to load. This is required and will cause all
+   * entities of the given type to be available in the selection
+   * and auto-complete
+   * @param type The type of the entity
+   */
+  setEntityType(type: EntityConstructor<E>) {
+    this.loading.next(true);
+    this.entityMapperService
+      .loadType<E>(type)
+      .then((entities) => {
+        this.allEntities = entities;
+        this.loading.next(false);
+        this.formControl.setValue(null);
+      })
+      .catch((error) => {
+        this.loggingService.warn(error);
+        this.loading.next(false);
+      });
```

Comment:
```suggestion
      });
```
There is no expected reason for this to throw an error. If it fails, staying indefinitely in loading mode seems as good as being empty. Errors thrown without catch are logged more explicitly with better stacktrace in sentry so I prefer not to have a catch here.